zkevm-circuits/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 //! # zk_evm
 
+#![allow(incomplete_features)]
+// Needed by DummyGadget in evm circuit
+#![feature(adt_const_params)]
```

Comment:
I'm not sure how unstable this feature is (in the sense of being problematic).  I would say that we should be careful to not use this in structs that are not temporary; so that we can remove it in the future (once the `DummyGadget` is no longer needed).